tests/client/test_client_task_meilisearch.py

```diff
@@ -165,3 +165,33 @@ def test_get_tasks_in_reverse(client):
     reverse_tasks = client.get_tasks({"reverse": "true"})
 
     assert reverse_tasks.results[0] == tasks.results[-1]
+
+
+def test_get_batches_default(client):
+    """Tests getting the batches."""
+    batches = client.get_batches()
+    assert len(batches.results) >= 1
+
+
+@pytest.mark.usefixtures("create_tasks")
+def test_get_batches_with_parameters(client):
+    """Tests getting batches with a parameter (empty or otherwise)."""
+    rev_batches = client.get_batches({"reverse": "true"})
+    batches = client.get_batches({})
+
+    assert len(batches.results) > 1
+    assert rev_batches.results[0].uid == batches.results[-1].uid
+
+
+def test_get_batch(client):
+    """Tests getting the details of a batch."""
+    batches = client.get_batches({"limit": 1})
+    batch = client.get_batch(batches.results[0].uid)
+    batch_dict = batch.__dict__
+    assert "uid" in batch_dict
+    assert "details" in batch_dict
+    assert "stats" in batch_dict
+    assert "duration" in batch_dict
+    assert "started_at" in batch_dict
+    assert "finished_at" in batch_dict
+    assert "progress" in batch_dict
```

Comment:
Because `Batch` is a class we know all the fields will be there. I think we could just check to make sure the `uid` is correct and be good. This will also allow for adding/removing fields in the future without updating tests.

```suggestion
    assert batch is not None
    assert batch.uid == batches.results[0].uid
```